frontend/src/app/trees/forests/tree-guidelines/tree-cutting/tree-cutting-rules/tree-cutting-rules.component.html

```diff
@@ -24,8 +24,10 @@
     <div class="usa-width-one-third tree-image-container">
       <img src="/assets/img/stump-height-sm.jpg" alt="stump height is measured from ground not snow cover" />
```

Comment:
Per my other comment about `figcaption` tags, each of these `img`/`figcaption` pairs should be contained within `figure` containers.